src/og/Globe.js

```diff
@@ -158,6 +159,13 @@ class Globe {
          */
         this._planetName = options.name ? options.name : PLANET_NAME_PREFIX + Globe._staticCounter;
 
+        /**
+         * quad tree type.
+         * @private
+         * @type {Number}
+         */
+        this._quadTreeType = options.quadTreeType;
```

Comment:
what happens when quadTreeType is not provided in options?

Reply:
I suppose quadTreeType is the required parameter. By default, it must be Earth quadtree.